### PR TITLE
update readme with description and setup instructions

### PR DESCRIPTION
**Pull request by jaykeHarrison**
update readme with description and install instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,104 @@
...
+https://jaykeharrison-nc-news.herokuapp.com/api
+
+## Setup Instructions
+
+### Installation Requirements
+
+- **Node.js**: v17.8.0 or later
+- **PostgreSQL**: v 12.10 or later
+
+### Cloning the repositry:
```
Comment (Rosem10) [inline README.md]: *repository (sorry to be THAT person)

File: README.md [hunk 2]
```diff
@@ -1,16 +1,104 @@
...
+```
+
+Then clone the repository:
+
+```
+$ git clone https://github.com/jaykeHarrison/be-nc-news.git
+$ cd nc-news
+```
+
+### Setup Dev & Test Enviroments
```
Comment (Rosem10) [inline README.md]: *Environment

**Review by Rosem10 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jaykeHarrison) [reply README.md]: fixed!

File: README.md (hunk 2 above)
Comment (jaykeHarrison) [reply README.md]: fixed!

**Pull request merged**